Segment-Nielsen-DCR/Classes/SEGNielsenDCRIntegration.m

```diff
@@ -24,9 +24,17 @@
 }
 
 
-NSString *returnAdLoadType(NSDictionary *src, NSString *key)
+NSString *returnAdLoadType(NSDictionary *options, NSDictionary *properties)
 {
-    NSString *value = [src valueForKey:key];
+    NSString *value;
+    if ([options valueForKey:@"adLoadType"]){
+        value = [options valueForKey:@"adLoadType"];
+    } else if ([options valueForKey:@"loadType"]) {
```

Comment:
Everything else looks good - but I don't think we should fall back to look for `loadType` in options, just in properties. `loadType` in options isn't specced - and I didn't put that in Android, so adding here would be a point of non-parity between the mobile SDKs. Thank!

Reply:
@brennan just removed that fallback.